release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -53,6 +53,10 @@ The installer now gathers serial console logs from the bootstrap and control pla
 
 For more information, see xref:../installing/installing-troubleshooting.adoc#installation-bootstrap-gather_installing-troubleshooting[Troubleshooting installation issues].
 
+[id="ocp-4-12-consistent-adddress-for-ironicapi"]
+==== Consistent IP address for Ironic API in bare-metal installations without a provisoning network  
+With this update, in bare-metal installations without a provisioning network, the Ironic API service is accessible through a proxy server. This proxy server provides a consistent IP address for the Ironic API service. If the Metal3 pod that contains `metal3-ironic` relocates to another pod, the consistent proxy address ensures constant communication with the Ironic API service.
```

Comment:
I am unsure about `Metal3` vs `metal3` too. 
I see both usages here https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.html#enabling-a-provisioning-network-after-installation_ipi-install-post-installation-configuration

Reply:
I think Metal3 is correct. The Kubernetes docs all reference Metal3 (and Ironic): https://metal3io.netlify.app/introduction.html
It is actually Metal³. But I am unsure if we want to replicate that superscript 3 in our docs. If it appears as Metal3 or metal3 in other places in our docs, then I think Metal3 is the correct one of these options.

Reply:
Sounds great! Thanks for confirming.